canonicalwebteam/image_template/__init__.py

```diff
@@ -48,6 +48,8 @@ def image_template(url, alt, width, height, hi_def, **attributes):
         extra_classes = attributes["extra_classes"]
         del attributes["extra_classes"]
 
+    lazy = attributes.get("lazy", True)
```

Comment:
sorry, 1 small change - you need to do `del attributes["lazy"]` after this line. Otherwise the markup gets a `lazy="False"` attribute!